app/javascript/gobierto_visualizations/modules/subsidies_controller.js

```diff
@@ -203,32 +203,32 @@ export class SubsidiesController {
 
     // Calculations box items
     const numberSubsidies = subsidiesData.length;
-    const sumSubsidies = d3.sum(amountsArray);
+    const sumSubsidies = d3.sum(amountsArray) || 0;
```

Comment:
What if you create a helper function to return all this calculations? Here's a obvious DRY

Reply:
Done!